src/editor/extensions/ext-arrows/ext-arrows.js

```diff
@@ -6,6 +6,9 @@
  * @copyright 2010 Alexis Deveria
  *
  */
+ import {
+  $id, $qa, $qq
+} from '../../../svgcanvas/utilities.js';
```

Comment:
I would like to make svgedit modules not dependant on internal source code of svgcanvas. Ultimately, I think we could have svgcanvas as a separate npm modules. 
However, these utilities functions have been attached to the smain vgcanvas object so you could reuse them only with the import of svgcanvas.js

``````
import SvgCanvas from '../svgcanvas/svgcanvas.js';
const {$id, $qa, isNullish, encode64, decode64, blankPageObjectURL} = SvgCanvas;
`````

In fact, if you have access to the svgcanvas runtime object (or the editor object), I didn't test but it should be possible to access to the function without import.

Reply:
Modified now please check it.